kpatch-build/kpatch-build

```diff
@@ -841,6 +855,9 @@ fi
 # shellcheck disable=SC1090
 source "$CONFIGFILE"
 
+[[ "$DISTRO" = openEuler ]] && [[ -z "$CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY" ]] && \
+	die "openEuler kernel doesn't have 'CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY' enabled"
```

Comment:
I'm guessing OpenEuler sets CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY by default, but is it absolutely necessary that it be turned on for this distro?  IOW, could one build conventional livepatches on this kernel?  Thanks.

Reply:
Q: Could one build conventional livepatches on this kernel?
A: yes, but absolutely necessary to need CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY set.

As we can see from the source code of the openEuler, without CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY,  there is even no klp_enable_patch function.
![wechat_20220428012215](https://user-images.githubusercontent.com/41671456/165584061-52a5b12e-f381-4469-9f0c-0efcc3ece927.png)

Q: Is CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY set by default?
A: It depends on the version of the kernel. From this [openEuler commit](https://mailweb.openeuler.org/hyperkitty/list/kernel@openeuler.org/message/MANDE5M5NQ3WRQSSW7WCIECNF67WPP75/), I  think they prefer to use their own strategy.